Sources/Public/Option.swift

```diff
@@ -10,10 +10,12 @@ extension Debug {
     public struct Option {
         public var prefix: String?
         public var indentSize: Int
+        public var isConsoleUsed: Bool
```

Comment:
I think, this property `isConsoleUsed` is decide output strategy which `print` or `os_log`.
So use `enum` property as like following is more intuitive for user?

```swift
public var output: Output

enum Output {
    case print
    case osLog
}
```

What do you think?

Reply:
Fix It!

I thought Print or osLog is nomatter for developer, but output-destination is important.
so, I make case-name  point output-destination

Reply:
Well, I understand your thought.

But depending on whether choice a `print` or `osLog` are effects destination **and format**.
Therefore, the intent to format is missing from the name `OutputDestination`.

I think that naming to represent whether `print` or `osLog` are not confusing for **iOS developers**. (However name the `Output` is abstract... make better to use `OutputStrategy` maybe)

What do you think?